neural_compressor/torch/algorithms/weight_only/autoround.py

```diff
@@ -26,7 +26,7 @@
 class AutoRoundQuantizer(Quantizer):
     def __init__(
         self,
-        weight_config: dict = {},
+        quant_config: dict = {},
```

Comment:
Empty dictionary is a dangerous default value https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Reply:
Set default value of `quant_config` here to None and pass an empty dictionary to AutoRoundProcessor if `quant_config` is None.